docs/content/preview/deploy/kubernetes/multi-zone/eks/helm-chart.md

```diff
@@ -93,39 +93,23 @@ $ eksctl create cluster \
 
 As stated in the Prerequisites section, the default configuration in the YugabyteDB Helm Chart requires Kubernetes nodes to have a total of 12 CPU cores and 45 GB RAM allocated to YugabyteDB. This can be three nodes with 4 CPU cores and 15 GB RAM allocated to YugabyteDB. The smallest AWS instance type that meets this requirement is `m5.2xlarge` which has 8 CPU cores and 32 GB RAM.
 
-### Create a storage class per zone
+### Create a storage class
 
 We need to ensure that the storage classes used by the pods in a given zone are always pinned to that zone only.
```

Comment:
Since the YAML no longer mentions zones, you should adjust the wording of this line to better explain what the chart does now.

Reply:
makes sense, done.